starknet-rpc-test/src/constants.rs

```diff
@@ -7,6 +7,7 @@ pub const CAIRO_1_ACCOUNT_CONTRACT_CLASS_HASH: &str =
     "0x35ccefcf9d5656da623468e27e682271cd327af196785df99e7fee1436b6276";
 pub const ERC20_CAIRO_0_CONTRACT: &str = "0x040e59c2c182a58fb0a74349bfa4769cbbcba32547591dd3fb1def8623997d00";
 pub const SEQUENCER_ADDRESS: &str = "0xdead";
+pub const UDC_ADDRESS: &str = "0x041a78e741e5af2fec34b695679bc6891742439f7afb8484ecd7766661ad02bf";
```

Comment:
need to update the readme + it's linked to this #1100 I think

Reply:
Which readme file?